easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.1-foss-2018b.eb

```diff
@@ -23,7 +23,6 @@ builddependencies = [
 
 dependencies = [
     ('GSL', '2.5'),
-    ('zlib', '1.2.11'),
```

Comment:
If GEMMA uses zlib, then zlib should be an explicit dependency and not implicit from the toolchain.